### ISSUE TITLE
can't convert SimpleUUID::UUID into String

### ISSUE THREAD

**Issue by tawkoo** [carbonfive/active_column#9]
Getting "can't convert SimpleUUID::UUID into String" when trying to create a row.

Environment:
Cassandra 0.8.4
Cassandra gem v0.11.4

By default cassandra gem was using simple_uuid 0.2 but looks like active_column has dependency with 0.1 so I had add gem 'simple_uuid', "~> 0.1.0 in Gemfile.

**Comment by mwynholds** [comment]
I looked at the latest Cassandra gem (0.12) and it appears to be using simple_uuid ~> 0.1.0.  So I added that same dependency to active_column (for now).  I also updated active_column to pull in cassandra/0.8 by default (although whatever you are pulling in in your app will win).  Can you try out HEAD and let me know if it works for you?  And if so, you can close this issue.

Thanks.

**Comment by tawkoo** [comment]
Still "can't convert SimpleUUID::UUID into String" error even it's using simple_uuid ~> 0.1.0.

I tried $cassandra.insert(:posts, 'bilal', {'title' => 'test', 'description' => 'test description'}) and that is working but not active column.

What would cause this?

**Comment by mwynholds** [comment]
Is your project up on github?  Can I clone it and see it fail locally?

**Comment by mwynholds** [comment]
I haven't heard from you in a while.  I'm going to close this, but please re-open it (or another issue) if you want to continue discussing it.

**Comment by shamne** [comment]
I have the same problem
cassandra 0.7
gem 'cassandra', '>= 0.9', :require => 'cassandra/0.7'
Tried to add simple_uuid ~> 0.1.0 to the gemfile, but no luck, still have the problem: 

ruby :010 > p = Post.new(:user_id=>"test")
 => #<Post:0x000001014e3048 @user_id="test"> 
ruby :011 > p.save
TypeError: can't convert SimpleUUID::UUID into String
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/columns.rb:93:in `initialize'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/columns.rb:93:in`new'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/columns.rb:93:in `_standard_insert_mutation'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/cassandra.rb:454:in`block in insert'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/cassandra.rb:454:in `each'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/cassandra.rb:454:in`collect'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/cassandra-0.12.0/lib/cassandra/cassandra.rb:454:in `insert'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/active_column-0.2/lib/active_column/base.rb:64:in`block in save'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/active_column-0.2/lib/active_column/base.rb:63:in `each'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/active_column-0.2/lib/active_column/base.rb:63:in`save'
    from (irb):11
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/railties-3.1.1/lib/rails/commands/console.rb:45:in `start'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/railties-3.1.1/lib/rails/commands/console.rb:8:in`start'
    from /Users/orion/Sites/orion/shamne/code/vendor/bundler_gems/ruby/1.9.1/gems/railties-3.1.1/lib/rails/commands.rb:40:in `<top (required)>'
    from script/rails:6:in`require'
    from script/rails:6:in `<main>'

**Comment by mwynholds** [comment]
Hi Shamne-

I'll re-open this and take a look as soon as I have a moment, which will hopefully be a little later this week.

..mike..

**Comment by mwynholds** [comment]
Ok, I figured this one out.  I think what is going on is that you have defined your "posts" column family to have a :string or :utf8 comparator.  But for the active timeline stuff to work, it needs to be a :timestamp comparator.  If you update your column family config it should work.

I looked back through the docs and it is not very clear.  I will update them.

Let me know if this fixes the problem.
